src/components/application_manager/src/application_impl.cc

```diff
@@ -347,6 +354,11 @@ void ApplicationImpl::set_is_media_application(bool is_media) {
   is_media_ = is_media;
 }
 
+void ApplicationImpl::set_audio_streaming_indicator(
+    mobile_apis::AudioStreamingIndicator::eType indicator) {
```

Comment:
@dtrunov @LevchenkoS I don't. const required not only for optimization. It explicatively tells that this variable should not be changed. Function parameters are same as local variables. And if you do not need to change them inside function, it is better use const for function parameters even for POD types ( but not const reference) .
ANy way, it is minor issue. Just remember about it